core/js/typespecs/functions.ts

```diff
@@ -88,6 +97,21 @@ const invokeFunctionNonIdArgsSchema = z.object({
     .describe(
       "Whether to stream the response. If true, results will be returned in the Braintrust SSE format.",
     ),
+  update_score: z
+    .object({
+      object_type: z.enum(["project_logs", "experiment"]),
+      object_id: z
+        .string()
+        .describe("The id of the container object you are logging to"),
+      row_id: z.string().describe("Id of the row to update"),
+      [ASYNC_SCORING_TOKEN_FIELD]: z
+        .string()
+        .describe("Token that launched the update"),
+    })
+    .nullish()
+    .describe(
+      "If specified, log the result of the function invocation as a score to the specified row. In this case, the function output must be a number between 0 and 1, inclusive",
```

Comment:
what happens if it returns a `Score` object?

Reply:
Good question. See comment in internal PR about what the correct output format should be for the function output.